test/expressions/bitshift/bitshiftLargerThanNumBits.chpl

```diff
@@ -0,0 +1,13 @@
+config const op = 0;
+
+var shift: int(64) = 128;
+var unsignedShift: uint(64) = 128;
+var shiftBy: int = 64;
+
+select op {
+when 0 do const foo = shift << shiftBy;
+when 1 do const foo = shift >> shiftBy;
+when 2 do const foo = unsignedShift << shiftBy;
+when 3 do const foo = unsignedShift >> shiftBy;
+}
```

Comment:
I can't see why non-param tests can be bundled together by making `shiftBy` a `config var`. but it is not a huge deal

Reply:
and indent the select block